docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging.adoc

```diff
@@ -639,6 +639,30 @@ This is done from the CLI:
     }
 }
 ----
+
+=== Critical analyzis of the broker
+
+When things go wrong on the broker, the critical analyzer acts as a safeguard shtting down the broker. +
+ If the response time goes beyond a configured timeout, the broker is considered unstable and an action will be taken to either shutdown the broker or halt the VM.
+For this, the critical analyzer measures the response time in:
+
+ * Queue delivery (add to the queue)
+ * Journal storage
+ * Paging operations
+
+You can configure the critical analyzer on the broker using the CLI
+To disable the critical analyzer, you cna do the follwoing CLI command:
+----
+[standalone@localhost:9990 /] /subsystem=messaging-activemq/server=default:write-attribute(name=critical-analyzer-enabled, value=false)
+{
+    "outcome" => "success",
+    "response-headers" => {
+        "operation-requires-reload" => true,
+        "process-state" => "reload-required"
+    }
+}
+----
+
```

Comment:
Since you'r adding this section might as well add a table or set of bullets listing the other options. I don't think showing them in an example op is needed though.